buildSrc/src/main/java/Gecko.kt

```diff
@@ -9,12 +9,12 @@ object Gecko {
     /**
      * GeckoView Version.
      */
-    const val version = "99.0.20220308092232"
+    const val version = "99.0.20220308094525"
 
     /**
      * GeckoView channel
      */
-    val channel = GeckoChannel.NIGHTLY
+    val channel = GeckoChannel.BETA
```

Comment:
Is there any docs related to this new constant?

Reply:
https://mozac.org/contributing/release-checklist

Reply:
Thanks!